From 3d63d6e03cc38bacec069169498f0ca679d56690 Mon Sep 17 00:00:00 2001
From: zhongfly <11155705+zhongfly@users.noreply.github.com>
Date: Tue, 16 Sep 2025 21:27:25 +0800
Subject: [PATCH] mpv: copy mpv-register and mpv-unregister helpers

---
 packages/mpv.cmake | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/packages/mpv.cmake b/packages/mpv.cmake
index 89df5a8..a7d9c70 100644
--- a/packages/mpv.cmake
+++ b/packages/mpv.cmake
@@ -77,5 +77,7 @@ ExternalProject_Add_Step(mpv copy-binary
     DEPENDEES strip-binary
     COMMAND ${CMAKE_COMMAND} -E copy <BINARY_DIR>/mpv.exe                           ${CMAKE_CURRENT_BINARY_DIR}/mpv-package/mpv.exe
     COMMAND ${CMAKE_COMMAND} -E copy <BINARY_DIR>/mpv.com                           ${CMAKE_CURRENT_BINARY_DIR}/mpv-package/mpv.com
+    COMMAND ${CMAKE_COMMAND} -E copy <SOURCE_DIR>/etc/mpv-register.bat              ${CMAKE_CURRENT_BINARY_DIR}/mpv-package/mpv-register.bat
+    COMMAND ${CMAKE_COMMAND} -E copy <SOURCE_DIR>/etc/mpv-unregister.bat            ${CMAKE_CURRENT_BINARY_DIR}/mpv-package/mpv-unregister.bat
     COMMAND ${CMAKE_COMMAND} -E copy ${MINGW_INSTALL_PREFIX}/bin/lua51.dll          ${CMAKE_CURRENT_BINARY_DIR}/mpv-package/lua51.dll
     COMMAND ${CMAKE_COMMAND} -E copy <BINARY_DIR>/mpv.pdf                           ${CMAKE_CURRENT_BINARY_DIR}/mpv-package/doc/manual.pdf
-- 
2.51.0.windows.1

